modules/liblc3/Kconfig

```diff
@@ -4,5 +4,6 @@
 config LIBLC3
 	bool "liblc3 Support"
 	depends on FPU
+	depends on NEWLIB_LIBC
```

Comment:
Since liblc3 is not (should not be) specifically tied to newlib, please use `REQUIRES_FULL_LIBC`.

```suggestion
	select REQUIRES_FULL_LIBC
```

Reply:
Done